lib/ship_quiet_logistics/documents/shipment_order.rb

```diff
@@ -94,6 +94,14 @@ def date_stamp
       def order_type
         'SO'
       end
+
+      def country_name(address)
```

Comment:
Does this belong in this PR? It seems out of scope.

Reply:
That's right, it shouldn't.
Originally, it was using `country_name` which is a decorator setup in Aday's project. We should use the bare behaviour of Spree instead: `country.name` as this gem is tested upon a dummy Spree project.